app/views/dashboard/index.html.haml

```diff
@@ -0,0 +1,20 @@
+.page-header
+  %h1.text-uppercase.text-center All Recipes
+
+.row
+  - @recipes.each_slice(3) do |recipe_two|
+    -recipe_two.each do |recipe|
+      .col-md-4.col-md-4.col-md-4.extramargin
+        .container-box.center-block
+          %div.img-rounded.center-block{style:"width:250px; height:200px; border:1px solid #000; margin-bottom:15px; background-color:#eee"}
```

Comment:
Since, we haven't installed the carrier waver gem, i just us that div so that it is the place holder for the image, Just to see the difference in layout